### ISSUE TITLE
Text prefill positioning

### ISSUE THREAD

**Issue by timdrysdale** [timdrysdale/parsesvg#1]
Text prefill is not working quite right - seems to be an offset of around 30mm in the y direction, which is either me rolling back an earlier change or never getting it right in the first place. Text fields are bang on - just text prefills are weird.

**Comment by timdrysdale** [comment]
So this is an issue with getting the correct translate value out of the SVG for text prefills.
```
     inkscape:label="textprefills"
     inkscape:groupmode="layer"
     id="layer1"
     transform="translate(0,-282)"
    <snip>
     y="283.22031"
```


```
     inkscape:groupmode="layer"
     id="layer4"
     inkscape:label="textfields">
    <rect
     transform="translate(0,-282.00001)"    
    <snip>
    y="284.1167
```
This should result in the text field and the prefill both being near the top of the 312pt page (A4 with a header bar. Except the textfield is fine and the prefill is down the page.

Debug printing shows this:-
```
Txfield Translate 0.000000 -282.000010
Prefill Translate 0.000000 0.000000
```
Therefore, the parsing of the svg to detect the (obviously there) translation of the textprefill is failing. A hacky fix is to edit your text prefill sections of the svg to avoid the need for a translate - but a better fix is to understand why the same parser function works for textfield and fails for prefill. Leaving this open until understood/fixed

**Comment by timdrysdale** [comment]
This is now fixed with [v0.3](https://github.com/timdrysdale/parsesvg/releases/tag/v0.3) It turned out that Inkscape sometimes uses the layer translate, and sometimes the object translate, but that I hadn't noticed until it decided to do it one way for text fields, and the other way for textprefills. I have now updated anchors, svg, textfields and textprefills to parse for the layer AND the object translate and apply it. Tests passing visual inspection.
